ios/COVIDSafePathsTests/ExposureManagerTests.swift

```diff
@@ -244,6 +245,29 @@ mn/1593460800-1593475200-00022.zip
     XCTAssertEqual(BTSecureStorage.shared.userState.remainingDailyFileProcessingCapacity, Constants.dailyFileProcessingCapacity)
   }
 
+  func testFilterOldKeysForSubmission() {
+    // Keys older than 350 hours should not be submitted
```

Comment:
How do we feel about including some of this comment on the test name? Would it be helpful for a failure error to read the context of the comment also?